### PR TITLE
Fix "Delete Answer Translation View"

### PR DESCRIPTION
**Pull request by badrihippo**
Fixed a typo in the template, and also updated the view to delete the related question translation as well (if it exists).

### CODE REVIEW

**Comment by badrihippo** [comment]
@jr-sawaliram you had mentioned something about not being able to delete. Will this fix the issue? If @subindevs is busy I think this is simple enough that one of the interns can also test and review.

File: dashboard/views.py [hunk 1]
```diff
@@ -2118,6 +2118,31 @@ class DeleteAnswerTranslation(BaseDeleteTranslation):
...
 
+    def delete(self, request, *args, **kwargs):
+        '''
+        Also delete the question object while we're about it
+        '''
+
+        obj = self.get_object()
+
+        if obj.is_published:
+            question_model = PublishedTranslatedQuestion
```
Comment (abhiprojectz) [inline dashboard/views.py:2129]: @badrihippo  Its working fine, but here `PublishedTranslatedQuestion` is not imported (nor in our present codebase).

Comment (badrihippo) [reply dashboard/views.py:2130]: Good catch! I've added the line now.

**Review by abhiprojectz — APPROVED** [review]

**Comment by jr-sawaliram** [comment]
Thanks @badrihippo, the translations are now deletable by users after they are converted to draft form by admin.  Trying to delete them at the admin interface gives server error.

Other outstanding problems appear to be caused by large image sizes:

Articles and article translations with large image sizes do not load at the admin interface. So there seems to be no way of deleting them. Hopefully when @abhiprojectz fixes #415 this problem will be avoided for future articles and answers. It's not clear what can be done with existing large image sizes. 

We have one test article translation (published) with large image size, not loading at the admin interface so cannot delete it.

**Comment by badrihippo** [comment]
> Articles and article translations with large image sizes do not load at the admin interface.

Maybe you can make a separate issue for this? If @abhiprojectz' fix doesn't work, that is.

**Comment by abhiprojectz** [comment]
> It's not clear what can be done with existing large image sizes.

@jr-sawaliram  existing images are less so then can be manually compressed.

**Comment by jr-sawaliram** [comment]
Issue #427 created.

**Pull request merged**